app/src/main/java/com/architectcoders/arquitectomarvel/model/personajes/Characters.kt

```diff
@@ -0,0 +1,20 @@
+package com.architectcoders.arquitectomarvel.model.personajes
+
+import com.google.gson.annotations.SerializedName
+
+data class Characters(
+    @SerializedName("attributionHTML")
```

Comment:
@joseantoniocampillo no vas a necesitar @SerializedName porque el nombre de los atributos de la clase son los mismos que las claves del JSON. Échale un vistazo a todas las demás data classes.

Reply:
Tengo costumbre de dejárselo (los hace el plugin json to kotlin class). Muchas veces requiero de que algunos campos se ignoren o añadir otros que tampoco quiere que se formen parte de la serialización a json. Para ello le añado al Gson que usa el Retrofit.Builder un metodo para eso quedando el Gson de esta forma GsonBuilder().excludeFieldsWithoutExposeAnnotation().create()
Cuando hago eso Gson solo ve los atributos anotado e ignore el resto.

Reply:
Disculpa @juancoob (y equipo) no estaba en lo cierto en mi respuesta anterior. Corrijo:
Les dejo la anotación que me pone el plugin por si quiero renombrar a posteriori algún atriguto y al ver la clase tengo más claro de que se trata.
Pero la anotación que se usa (la use en varios proyectos, pero no la recordaba) para que se ignoren los atributos que no la tienen cuando se la pasa al builder el excludeFieldsWithoutExposeAnnotation() method es la anotación @Expose.